NetworkLayer/Model/Services/IdItemService.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  IdAdService.swift
+//  NetworkLayer
+//
+//  Created by Nicolas Demange on 18/12/2022.
+//
+
+import Foundation
+
+// MARK: - Protocol
+
+protocol ItemService {
```

Comment:
```suggestion
protocol CategoryService {
```
Attention au mot Item, ça ne veut rien dire sans context.